### PR TITLE
PR for Kickoff Branch

### PR DESCRIPTION
**Pull request by tadeuasarro**
### Main features:
- All HTML elements created using ES6;
- Switch between "contact" and "menu" tabs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,5 +7,26 @@
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (trekab) [inline package.json]: Add the necessary run commands like `build`, `development` should be in `package.json` inside `scripts`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const element = document.createElement('div');
+  element.id = 'content';
+
+  element.appendChild(nav);
+  element.appendChild(main);
+  element.appendChild(footer);
+
+  return element;
+}
+
```
Comment (trekab) [inline src/index.js]: Make use of ES6+ syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) i.e. arrow functions.

File: src/main/tabs/contact_tab.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    smallContainer.innerHTML = '';
+    smallContainer.appendChild(contactTab());
+  });
+
+  menu.addEventListener('click', () => {
+    const smallContainer = document.getElementById('small_container');
+    smallContainer.innerHTML = '';
+    smallContainer.appendChild(menuTab);
+  });
+
```
Comment (trekab) [inline src/main/tabs/contact_tab.js]: You have this same logic repeated in the `menu_tab.js` file. It is a good practice to keep your code [DRY](https://dzone.com/articles/software-design-principles-dry-and-kiss). Please add this logic in the `index.js` file as the requirements indicated.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Coxinha's Heaven</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href='../node_modules/tabsy-css/dist/tabsy.css' rel='stylesheet' type='text/css'>
```
Comment (trekab) [inline dist/index.html]: Please remove this('../node_modules/tabsy...') unnecessary CSS import. It is not adding any value to your project.

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import './styles.css';
+import nav from './navbar';
+import main from './main/main';
+import footer from './footer';
+
+function component() {
+  const element = document.createElement('div');
+  element.id = 'content';
```
Comment (trekab) [inline src/index.js]: This element wasn't meant to be created dynamically from JS. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @tadeuasarro! Well done so far. You have just a few adjustments to make to complete this project:
- [ ] Given that the tabbed menu is the main feature of this project, putting it at the bottom of the page makes it easy to miss and the tab should __wipe out__ the current page contents and then run the correct ‘tab module’ to populate it again. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
- [ ] Check the comments in the code below for the rest of the feedback.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by lucasmazo23 — APPROVED** [review]
## Project approved 🚀
### Great job!
### Your project looks great!
### You can go ahead and merge this pull request!
### Happy coding! 💻
## Lucas Mazo

**Pull request merged**